.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  rubocop:
```

Comment:
Rubocop is linter for ruby code, all we need in this file is to enable the [style lint](https://github.com/microverseinc/linters-config/tree/master/css)